x-pack/plugins/observability_solution/apm/ftr_e2e/cypress/e2e/alerts/alerts_table.cy.ts

```diff
@@ -90,11 +89,7 @@ describe('Alerts', () => {
       // Check that the "Alerts" table is loaded
       cy.wait(2000);
       cy.visitKibana('/app/apm/services/opbeans-java/alerts');
-      cy.getByTestSubj('o11yGetRenderCellValueLink').first().click({ force: true });
-      cy.getByTestSubj('alertsFlyout').should('exist');
```

Comment:
Q: Why are those checks removed? Do we want to check only the alerts table without the flyout?

Reply:
I deleted those checks because there was not link in the APM table page to open that flyout, @benakansara created this test, maybe she can clarify what is that link

Reply:
I think the alert reason link is broken after some refactoring of alerts table. Now we need to provide `cellContext` as done in this [PR](https://github.com/elastic/kibana/pull/186563) for similar issue to have the link. It would be nice to have these checks in test. It indeed caught a bug in this case.

Reply:
Thanks @benakansara, I'll take a look

Reply:
created https://github.com/elastic/kibana/issues/188921 to fix the alert reason link